### PR TITLE
AC Votes Eyeshade

### PR DESCRIPTION
**Pull request by husobee**
updates to add ac vote retrieval from kafka, and insertion into the votes table

### CODE REVIEW

File: eyeshade/workers/acvote.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+const { votesId } = require('../lib/queries.js')
+const { voteType } = require('../lib/vote.js')
+
+const voteTopic = process.env.ENV + '.payment.vote'
+
+module.exports = (runtime, callback) => {
+  runtime.kafka.on(voteTopic, async (messages) => {
+    const client = await runtime.postgres.connect()
+    const now = new Date()
+    const date = [
```
Comment (corymcdonald) [inline eyeshade/workers/acvote.js:10]: The JS Date library has a lot of weirdness, especially around timezones is there any worry here? `Moment.js` might be a better option? Thoughts?

File: eyeshade/workers/acvote.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+            publisher,
+            surveyorId
+          ])
+        }
+      } catch (e) {
+        await client.query('ROLLBACK')
+        runtime.captureException(e, { extra: { topic: voteTopic } })
+        throw e
+      }
+      await client.query('COMMIT')
```
Comment (corymcdonald) [inline eyeshade/workers/acvote.js:65]: When this raises an exception due to a lock or db constraint do we capture that error somewhere?

File: eyeshade/workers/acvote.js (hunk 1 above)
Comment (husobee) [reply eyeshade/workers/acvote.js]: https://github.com/brave-intl/bat-ledger/pull/822/commits/8264f629ed6b80b9459222d3da822b21226ca11b <- addressed here, also updated suggestions to match.

File: eyeshade/workers/acvote.js (hunk 2 above)
Comment (husobee) [reply eyeshade/workers/acvote.js]: addressed in https://github.com/brave-intl/bat-ledger/pull/822/commits/8264f629ed6b80b9459222d3da822b21226ca11b logging out if there were errors from commit/rollback.

**Pull request merged**